piet/src/render_context.rs

```diff
@@ -34,6 +45,12 @@ pub trait RenderContext {
     /// Parameters for the style of stroke operations.
     type StrokeStyle;
 
+    type FBuilder: FontBuilder<Out = Self::F>;
+    type F: Font;
```

Comment:
Any reason this type is just `F` instead of `Font`? I know `Font: Font` looks weird, but I imagine having just `F` will be confusing down the line and will make reading code harder than it needs to be.

Reply:
I think you're right. I wasn't sure whether there would be namespace collisions, but calling both Font does make sense. I think that's probably out of scope for this PR though, happy to take the rename in another one.